src/customers/service/customers.service.ts

```diff
@@ -0,0 +1,27 @@
+import { BadRequestException, Injectable, NotFoundException } from "@nestjs/common";
+import { UUID } from "crypto";
+import { Customer } from "../domain/customer.entity";
+import { CustomerRepository } from "../repository/customer.repository";
+
+
+@Injectable()
+export class CustomerService{
+
+    constructor(private readonly customerRepository: CustomerRepository){}
+
+    async getById(id: UUID): Promise<Customer> {
+        try {
+            return await this.customerRepository.getById(id);
```

Comment:
**suggestion:** if a function is already declared as async and you return a promise, there is no reason for you to add the await statement near that return

Reply:
Yes, it makes sense. Applied the suggestion. 
It's like waiting for the promise to solve in order to convert it back to a promise again because of the return type of the function, right? That being unnecessary.

Reply:
Yes it is unnecessary since the promise will be awaited somewhere else at the top level anyway (e.g.: controller response on the http request)